cmd/cspc-operator/app/storagepool_create.go

```diff
@@ -207,7 +207,7 @@ func getDeployLabels(csp *apis.CStorPoolInstance) map[string]string {
 	return map[string]string{
 		string(apis.CStorPoolClusterCPK): csp.Annotations[string(apis.CStorPoolClusterCPK)],
 		"app":                            "cstor-pool",
-		"openebs.io/cstor-pool":          csp.Name,
+		"openebs.io/cstor-poolinstance":  csp.Name,
```

Comment:
will change this to `openebs.io/cspi` or `openebs.io/cstor-pool-instance` ? @vishnuitta

Reply:
openebs.io/cstor-pool-instance is good.

Reply:
Done